core/src/main/java/org/apache/iceberg/TableProperties.java

```diff
@@ -246,9 +246,6 @@ private TableProperties() {}
   public static final String WRITE_PARTITION_SUMMARY_LIMIT = "write.summary.partition-limit";
   public static final int WRITE_PARTITION_SUMMARY_LIMIT_DEFAULT = 0;
 
-  public static final String MANIFEST_LISTS_ENABLED = "write.manifest-lists.enabled";
```

Comment:
Should we deprecate this constant so that people see that it no longer works?

Reply:
good point, will do that